http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoServiceSpec.scala

```diff
@@ -499,7 +500,7 @@ class LeonardoServiceSpec
                                                     computeDAO,
                                                     directoryDAO,
                                                     iamDAO,
-                                                    projectDAO,
+                                                    projectDAO, MockWelderDAO,
```

Comment:
I'm going to run scalaFmt at the end, I can't be bothered to format all the things IntelliJ does with the refactor shortcuts as I go